specs/_features/eip7594/das-core.md

```diff
@@ -64,18 +66,13 @@ The following values are (non-configurable) constants used throughout the specif
 | `NUMBER_OF_COLUMNS` | `uint64(CELLS_PER_EXT_BLOB)` (= 128) | Number of columns in the extended data matrix |
 | `MAX_CELLS_IN_EXTENDED_MATRIX` | `uint64(MAX_BLOBS_PER_BLOCK * NUMBER_OF_COLUMNS)` (= 768) | The data size of `ExtendedMatrix` |
 
-### Networking
-
-| Name | Value | Description |
-| - | - | - |
-| `DATA_COLUMN_SIDECAR_SUBNET_COUNT` | `128` | The number of data column sidecar subnets used in the gossipsub protocol |
-
 ### Custody setting
 
 | Name | Value | Description |
 | - | - | - |
 | `SAMPLES_PER_SLOT` | `8` | Number of `DataColumnSidecar` random samples a node queries per slot |
-| `CUSTODY_REQUIREMENT` | `4` | Minimum number of subnets an honest node custodies and serves samples from |
+| `NUMBER_OF_CUSTODY_GROUPS` | `128` | Number of column groups available for nodes to custody |
```

Comment:
Yes, a custody group is a collection of columns, but calling it a "column group" is a inconsistent.
```suggestion
| `NUMBER_OF_CUSTODY_GROUPS` | `128` | Number of custody groups available for nodes to custody |
```

Reply:
Yeah, maybe I missed that during the rebase